packages/commonwealth/client/scripts/views/pages/LaunchToken/steps/TokenInformationStep/TokenInformationForm/TokenInformationForm.tsx

```diff
@@ -68,6 +70,24 @@ const TokenInformationForm = ({
 
   const { mutateAsync: createToken } = useCreateTokenMutation();
 
+  const debouncedSearchTerm = useDebounce<string | undefined>(tokenName, 500);
+
+  const { data: tokensList } = useFetchTokensQuery({
+    cursor: 1,
+    limit: 50,
```

Comment:
actually if there will be at least 1 item in response, it means that the token name is occupied. So we don't need 50 here, right?

Reply:
I thought about that but it may break in some scenarios like if a token with name `tkn-1-100` exists and user enters input `tkn-1-10`, the results will contain 1 item (maybe more) - so looking for exact matches.